doc/changelog.rst

```diff
@@ -17,6 +17,8 @@ PyMongo 4.16 brings a number of changes including:
 - Fixed return type annotation for ``find_one_and_*`` methods on :class:`~pymongo.asynchronous.collection.AsyncCollection`
   and :class:`~pymongo.synchronous.collection.Collection` to include ``None``.
 - Added support for NumPy 1D-arrays in :class:`bson.binary.BinaryVector`.
+- Fixed a bug in :class:`~pymongo.encryption.ClientEncryption`: ClientEncryption
```

Comment:
suggested wording: "Prevented ClientEncryption from loading the crypt shared library to fix "MongoCryptError: An existing crypt_shared library is loaded by the application" unless the linked library search path is set."